backend/FactoryOps.Api/Database/Models/CreateItemModel.cs

```diff
@@ -0,0 +1,8 @@
+namespace FactoryOps.Api.Database.Models;
+
+public record CreateItemModel(
+	int Group,
+	string Title,
+	string StartTime,
+	int Length,
```

Comment:
Maybe let's change this variable to hours, or at least lengthInHours

Reply:
great idea <3